traffic_stats/traffic_stats.go

```diff
@@ -229,6 +274,16 @@ func main() {
 	hupChan := make(chan os.Signal, 1)
 	signal.Notify(hupChan, syscall.SIGHUP)
 
+	dataExporters := []DataExporter{}
+
+	if config.KafkaConfig.Enable == true && c != nil {
+		dataExporters = append(dataExporters, c)
+	}
+
+	if config.DisableInflux == false {
```

Comment:
This can be simplified to `if !config.DisableInflux`

Reply:
I'd like to point out once again that with this change our configuration file for TO will have both `disableInflux` and `influxEnabled` - and I don't think that's ideal

Reply:
This isn't the TO config file -- this is the Traffic Stats config file.